annotations/security-schemes/README.md

```diff
@@ -11,14 +11,26 @@
 
 (see full [description](oauth-2-custom-settings.raml))
 
-### What's included?
+### What's included
 
 This directory contains the following RAML `AnnotationTypeDeclaration` fragment:
+
 - oauth-2-custom-settings.raml
 
 and the following example:
+
 - oauth-2-custom-settings-example.raml
 
 ### Which tools support this
 
 - [mulesoft/api-console (v5.0.x)](https://github.com/mulesoft/api-console)
```

Comment:
Do you want to bump the version here or maybe say `> v5.0`?

Reply:
Nah, it's still valid for 5.x, even though we don't support it.

Reply:
Ah, I know what you mean. I'll fix that.

Reply:
Fixed